src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -53,7 +53,7 @@ public class Feature
         private static readonly Feature __findAndModifyWriteConcern = new Feature("FindAndModifyWriteConcern", new SemanticVersion(3, 2, 0));
         private static readonly Feature __findCommand = new Feature("FindCommand", new SemanticVersion(3, 2, 0));
         private static readonly Feature __geoNearCommand = new Feature("GeoNearCommand", new SemanticVersion(1, 0, 0), new SemanticVersion(4, 1, 0, ""));
-        private static readonly Feature __groupCommand = new Feature("GroupCommand", new SemanticVersion(1, 0, 0), new SemanticVersion(4, 0, 0, "rc1"));
+        private static readonly Feature __groupCommand = new Feature("GroupCommand", new SemanticVersion(1, 0, 0), new SemanticVersion(4, 1, 1, "rc1"));
```

Comment:
Sorry, I was unclear here!  We don't know the pre-release version where the command was removed so for now let's assume it was removed in 4.1.1 with a pre-release string of `""`, which effectively says "any pre release version of 4.1.1" (see [here ](https://github.com/mongodb/mongo-csharp-driver/blob/21aff044981244cf1158c6d8e01e6bf542a3ed17/src/MongoDB.Driver.Core/Core/Misc/SemanticVersion.cs#L107)for all the gory details).